src/main/docker/README.md

```diff
@@ -0,0 +1,19 @@
+# Building and running a Jenkins master with Docker
+
+This Jenkins master image is configured to be used by integration tests, see the [main README](../../../../../README.md)
+
+
+Build the image
+```bash
+docker build -t jenkins-rest/jenkins .
```

Comment:
Would it make sense to use testcontainers to build the image instead of managing it on the command line?

Reply:
yes, a better build integration has previously been suggested (https://github.com/cdancy/jenkins-rest/issues/72#issuecomment-548005038)
I had suggested to do this afterwards, considering this PR as a 1st step to provide environment setup info and to run integration tests on Travis

Reply:
yeah I'm fine with this if only to get proper integ tests running which is a huge help